### PR TITLE
YD-380 Greatly simplified the Gradle scripts

### PR DESCRIPTION
**Pull request by Bert-R**

### CODE REVIEW

File: dbinit/build.gradle [hunk 1]
```diff
@@ -1,71 +1,48 @@
...
 }
 
 dependencies {
 	compile project(":core")
-	compile("org.springframework.boot:spring-boot-starter-data-jpa")
-	compile("org.springframework.batch:spring-batch-core:3.0.7.RELEASE")
-    compile("org.liquibase:liquibase-core:3.5.3")
-
-	runtime("org.hsqldb:hsqldb")
+	runtime "org.hsqldb:hsqldb"
```
Comment (aristotelos) [inline dbinit/build.gradle]: Can't HSQLDB be included in the root build.gradle?

**Comment by aristotelos** [comment]
Looks great, added a single question.

**Comment by Bert-R** [comment]
Where? I don't see your question.

File: dbinit/build.gradle (hunk 1 above)
Comment (aristotelos) [inline dbinit/build.gradle]: OK, agreed

**Pull request merged**